src/reference-manual/statements.qmd

```diff
@@ -435,20 +466,20 @@ and explicitly incrementing the log probability function, as in
 target += normal_lpdf(y | mu, sigma);
 ```
 
-The sampling statement drops all the terms in the log probability
+The distribution statement drops all the terms in the log probability
 function that are constant, whereas the explicit call to
 `normal_lpdf` adds all of the terms in the definition of the log
 normal probability function, including all of the constant normalizing
 terms.  Therefore, the explicit increment form can be used to recreate
 the exact log probability values for the model.  Otherwise, the
-sampling statement form will be faster if any of the input expressions,
+distribution statement form will be faster if any of the input expressions,
 `y`, `mu`, or `sigma`, involve only constants, data
 variables, and transformed data variables.
 
 
 ### User-transformed variables {-}
 
-The left-hand side of a sampling statement may be a complex
+The left-hand side of a distribution statement may be a complex
```

Comment:
This is new since we have complex numbers, but I'd rather have this say "arbitrary expression (of compatible type)".